frontend/js/reducers/battle-reducer.js

```diff
@@ -0,0 +1,21 @@
+import { GET_BATTLE } from '../constants';
+
+const initialState = {
+  battle: {},
+  isLoading: true,
+};
+
+const battleReducer = (state = initialState, action) => {
```

Comment:
In case you don't know, you can write this like:

```suggestion

const battleReducer = (state = { battle: {}, isLoading: true }, action) => {
```

But feel free to use it as you prefer, it's just a note :)